test/elixir_analyzer/exercise_test/common_checks/compiler_warnings_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule ElixirAnalyzer.ExerciseTest.CommonChecks.CompilerWarningsTest do
+  use ExUnit.Case
+  alias ElixirAnalyzer.ExerciseTest.CommonChecks.CompilerWarnings
+
+  test "Implementing a protocol doesn't trigger a compiler warning" do
+    filepath = "test_data/clock/lib/clock.ex"
+    assert CompilerWarnings.run(filepath, nil) == []
+  end
```

Comment:
How can I get this test to fail? Removing `consolidate_protocols: false,` form `mix.exs` doesn't work.

Reply:
Nevermind, I was editing `elixir/mix.exs` 🤦‍♀️

Reply:
Hahaha, yeah, that won't work :)